### DISCUSSION TITLE
book/kubernetes-handbook/architecture/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/rootsongjc.github.io#348] [Announcements]
# book/kubernetes-handbook/architecture/

本章介绍 Kubernetes 的架构设计。

https://jimmysong.io/book/kubernetes-handbook/architecture/

**Comment by qidaneix** [comment]
nice!

**Comment by JinlingTest** [comment]
整理的基础理论知识 好

**Comment by jackblack369** [comment]
great job, thanks for sharing!
